### PR TITLE
RC7 -> RC8

### PR DESCRIPTION
**Pull request by martinvol**
Brought already deployed RC7 commits to RC8 to prevent rollback on release, and confusing contract updates.

### CODE REVIEW

File: packages/celotool/package.json [hunk 1]
```diff
@@ -6,17 +6,16 @@
...
+    "@celo/base": "1.5.3-dev",
+    "@celo/connect": "1.5.3-dev",
+    "@celo/contractkit": "1.5.3-dev",
     "@celo/env-tests": "1.0.0",
     "@types/ethereumjs-util": "^5.2.0",
-    "@celo/explorer": "2.0.1-dev",
-    "@celo/governance": "2.0.1-dev",
-    "@celo/identity": "2.0.1-dev",
-    "@celo/network-utils": "2.0.1-dev",
-    "@celo/utils": "2.0.1-dev",
```
Comment (aaronmgdr) [inline packages/celotool/package.json]: moving the @celo/** back to 1.5 is probably not right

File: packages/faucet/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (aaronmgdr) [inline packages/faucet/package.json]: facaucet was move to its own repo

**Review by alecps — APPROVED** [review]
Thanks!!

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
requesting that old versions are not brought back. same with packages/ folders

File: packages/sdk/wallets/wallet-walletconnect/lib/test/common.d.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/// <reference types="node" />
```
Comment (aaronmgdr) [inline packages/sdk/wallets/wallet-walletconnect/lib/test/common.d.ts]: all the wallets/wallet-connect/** should be removed

**Review by aaronmgdr — APPROVED** [review]

**Comment by arthurgousset** [comment]
Thanks for the speedy review @aaronmgdr and @alecps 🙏

**Pull request merged**